docker-compose-es.yml

```diff
@@ -0,0 +1,59 @@
+version: '3'
+
+services:
+
+  postgres:
+    image: postgres
+    container_name: pgsql-dev
+    networks: 
+      - redarc
+    environment:
+      POSTGRES_PASSWORD: test1234
+    volumes:
+      - pgredarc01:/var/lib/postgresql/data
+    ports:
+      - "5432:5432"
+
+  redarc:
+    build:
+      context: https://github.com/Yakabuff/redarc.git
```

Comment:
I had to change context to `.` to get the container to run properly but it should be fine once I merge it right?